source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -15,6 +15,42 @@ content: |
      `Adding permissions to a user <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_change-permissions.html>`__
      to set these permissions.
 
+  .. important:: Use IAM roles
```

Comment:
[suggestion] minor - "Use IAM roles instead" would be a slightly clearer note title to me on first read - It took some reading to be sure this meant completely replace IAM user usage with roles (at least as far as your app is concerned).

I was going to suggest more language to make it clear you use roles instead of users, but I see there's a mobile app case where it looks like you would still create a user, then map it to a role.

Reply:
To clarify - when I was reading, I was wondering "do I still create a user in step 1, then create a role for my app to use in addition? or do I read this note first, then essentially replace "user" with "role" every time I see it in these steps?"

Reply:
Updated to "Authenticate with IAM Roles in Production", to try and more clearly say in note title exactly what you are using IAM role for.

Context: As mentioned in your message, it's tricky because you don't necessarily replace users with roles (a valid deployment can still require both AFAICT). I think we are trying to outsource most of the actual, "whats a relationship between a user and a role" to the AWS docs, and focus on "how do I use temporary credentials to assume a role" in this guide.